package.json

```diff
@@ -118,6 +118,7 @@
     "mathjs": "^7.0.0",
     "moment": "^2.24.0",
     "nerdamer": "^1.1.4",
+    "puppeteer": "^4.0.1",
```

Comment:
This should be in the `devDependencies`.

Reply:
Done.